src/patternfly.module.js

```diff
@@ -7,9 +7,10 @@
 angular.module('patternfly', [
   'patternfly.autofocus',
   'patternfly.card',
+  'patternfly.charts',
```

Comment:
I don't think we want this here. This forces everyone who is using angular-patternfly to also pull in c3/d3 which they may not want.